CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Bug-fixes within the same version aren't needed
 
 -->
 
+## Master
+
+* replacing the status bar spinner to VS Code spinner
```

Comment:
you might want to add user name at the end...

Reply:
Fixed